pkg/controller/command/presentproof/models.go

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package presentproof
+
+import "github.com/hyperledger/aries-framework-go/pkg/client/presentproof"
+
+// DeclinePresentationArgs model
+//
+// This is used when the presentation needs to be rejected
+//
+type DeclinePresentationArgs struct {
+	// PIID protocol state machine identifier
+	PIID string `json:"piid"`
```

Comment:
`omitempty` is not needed here and all json definitions below?

Reply:
No, we do not need `omitempty`. I need to Unmarshal incoming data/arguments, but I do not need to Marshal them.